ext/app/pipe/bootstrap.js

```diff
@@ -15,6 +15,9 @@ function bootstrapGrist(options) {
   if (seedFile) {
     window.seedFile = new URL(seedFile, window.location.href);
   }
+  if (options.initialData) {
+    window.initialData = options.initialData;
```

Comment:
After this PR, I'm going to start pushing for some namespacing, such as `window.gristStatic.<propertyName>`, to collect the bits and pieces we pass around via the window global. Doesn't feel right to put cleaning that up on you since I'm the one who introduced the pattern.